mantidproject/mantid
# Update FDA to handle groupasymmetries, multiperiod and co-added data

**Description of work.**

This updates the Frequency Domain Analysis 2 GUI to handle group asymmetries, multiperiod data and co-added data correctly. 

**Report to:** anthony. 

**To test:**

 * Open the new GUI and load some multiperiod data i.e.(HIFI84447). 
 * Check that the correct options appear in the Transform widget.
 * Change the periods to use on the home tab, click update all (this should happen automatically after #24835 ) and check again that the options are correct.
  * Try to Co-Add some data and check that the correct options appear in the transform tab



Partially Fixes #24748  


*This does not require release notes* because This GUI is not yet user facing

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
